### DISCUSSION TITLE
Support for HTTP HEAD requests

### DISCUSSION THREAD

**Discussion by hailspuds** [microfeed/microfeed#18] [Ideas] [answered]
This might be less an idea, and more a bug? Or maybe my implementation? 

Anyway, I was running my test feed through a few validators, and they are whinging about HTTP HEAD requests.

I'm not sure if this is an issue with my Cloudflare setup rather than Microfeed itself? 

<img width="954" alt="Screen Shot 2022-12-30 at 12 38 57 pm" src="https://user-images.githubusercontent.com/851118/210026169-84795031-1666-4a2e-9f32-9cf4ee49ccea.png">

**Comment by wenbinf** [comment, marked as answer]
This should be fixed by https://github.com/microfeed/microfeed/pull/21 :)

Thanks!
